pgjdbc/src/test/java/software/aws/rds/jdbc/postgresql/ca/AwsIamAuthenticationIntegrationTests.java

```diff
@@ -0,0 +1,48 @@
+package software.aws.rds.jdbc.postgresql.ca;
+
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.postgresql.PGProperty;
+import org.postgresql.util.PSQLException;
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+@Disabled
+public class AwsIamAuthenticationIntegrationTests {
+
+  private static final String CONNECTION_STRING_PREFIX = "jdbc:postgresql:aws://";
```

Comment:
"jdbc:postgresql:aws://" -> "jdbc:postgresql://" ?
Let's make test more generic, not depend on AWS driver

Reply:
Makes sense. Updated.